### DISCUSSION TITLE
Is there an easy way to add snippets?

### DISCUSSION THREAD

**Discussion by mainendra** [nvim-mini/mini.nvim#180] [Q&A] [answered]
How to add snippets until `mini.snippets` is ready.
```
    -- Snippets
    {'L3MON4D3/LuaSnip'},
    {'rafamadriz/friendly-snippets'},
```

**Comment by echasnovski** [comment, marked as answer]
Well, yes, by using some other plugins. These are good, I use 'L3MON4D3/LuaSnip' myself, but with manually managed snippets.

If you need more examples, here is [my Luasnip config](https://github.com/echasnovski/nvim/blob/master/lua/ec/configs/luasnip.lua) (has some hacks for showing if there is available snippet in Insert mode, as 'mini.completion' is not intended to show snippets) and here is [my snippet collection](https://github.com/echasnovski/nvim/tree/master/misc/snippets) (sourced from various sources, 'rafamadriz/friendly-snippets' being one of them).

**Reply by mainendra** [reply]
Can you suggest something for `friendly-snippets` instead of manually managed snippets?

**Reply by echasnovski** [reply]
> Can you suggest something for `friendly-snippets` instead of manually managed snippets?

I am not sure I follow you here. The [rafamadriz/friendly-snippets](https://github.com/rafamadriz/friendly-snippets) is already a curated collection of snippets for various file types. You [install it](https://github.com/rafamadriz/friendly-snippets#install), [install LuaSnip](https://github.com/L3MON4D3/LuaSnip#install), make it [know about 'friendly-snippets'](https://github.com/L3MON4D3/LuaSnip#add-snippets) and [make some mappings](https://github.com/L3MON4D3/LuaSnip#keymaps). This should be enough to use them.

**Reply by mainendra** [reply]
Oh yes, it's working. Thanks. Got confused as it was not showing up in auto complement options. It's was just `!` sign.
